ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/mac-security.j2

```diff
@@ -22,7 +22,7 @@ mac security
 {%         for connection_key in profile.connection_keys | arista.avd.natural_sort('id') %}
 {%             if connection_key.encrypted_key is arista.avd.defined %}
 {%                 set key_cli = "key " ~ connection_key.id ~ " 7 " ~ connection_key.encrypted_key | arista.avd.hide_passwords(hide_passwords) %}
-{%                 if connection_key.fallback is arista.avd.defined %}
+{%                 if connection_key.fallback is arista.avd.defined and connection_key.fallback %}
```

Comment:
While your version is correct, we do it like this in AVD - means it has to be defined and value `True`
```suggestion
{%                 if connection_key.fallback is arista.avd.defined(true) %}
```